package.json

```diff
@@ -25,7 +25,7 @@
 	],
 	"devDependencies": {
 		"@lodder/grunt-postcss": "^3.1.1",
-	  	"@playwright/test": "1.38.1",
+	  	"@playwright/test": "1.32.0",
```

Comment:
Note to self: we should update the version in Gutenberg as it's wholly out of date

Reply:
I've got a PR for it, but there's a regression in a recent version which breaks a few tests. Ref: https://github.com/WordPress/gutenberg/pull/54051#issuecomment-1698886612.